translator/translate/otel/receiver/adapter/translators.go

```diff
@@ -159,10 +161,27 @@ func fromInputs(conf *confmap.Conf, validInputs map[string]bool, baseKey string)
 					continue
 				}
 			}
+
 			cfgKey := common.ConfigKey(baseKey, inputName)
-			if measurement := common.GetArray[any](conf, common.ConfigKey(cfgKey, common.MeasurementKey)); measurement != nil && len(measurement) == 0 {
+
+			hasMeasurements := true
+			hasOnlyNonAdaptedMetrics := false
```

Comment:
I would rename this to something simpler like `noTelegrafReceivers`
```suggestion
			noTelegrafReceivers := false
```